index.html

```diff
@@ -14,16 +14,29 @@
     <link rel="stylesheet" href="./styles.css" />
   </head>
   <body>
-    <div class="toolbar">
-      <nav class="nav-bar">
-        <a href="#"><img src="./images/logo.png" alt="logo" /></a>
-        <ul class="sid-cons">
-          <li class="p">Portoflio</li>
-          <li class="a">About</li>
-          <li class="c">Contact</li>
-          <li class="material-symbols-outlined">menu</li>
+    <header class="header">
+      <nav class="navbar">
+        <a href="#"><img src="./images/logo.png" alt="logo"  class="logo-text"/></a>
+        <ul class="nav-menu">
+          <li class="nav-item nav-item1">
+            <a href="#cards" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item nav-item2">
+            <a href="#about" class="nav-link">About</a>
+          </li>
+          <li class="nav-item nav-item3">
+            <a href="#contact" class="nav-link">Contact</a>
+          </li>
         </ul>
+        <div class="hamburger">
+          <span class="bar"></span>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that the portfolio is responsive on a mobile device